ddtrace/span.py

```diff
@@ -258,6 +258,10 @@ def set_tag(self, key, value=None):
         except Exception:
             log.warning("error setting tag %s, ignoring it", key, exc_info=True)
 
+    def _set_str_tag(self, key, value):
```

Comment:
Might be worth adding this to the integration checklist and maybe to trace_utils.py so we can make a toolbox of methods we want people to use when writing integrations